osu.Game/Screens/Multi/Header.cs

```diff
@@ -96,7 +67,7 @@ public Header(ScreenStack stack)
         [BackgroundDependencyLoader]
         private void load(OsuColour colours)
         {
-            screenType.Colour = colours.Yellow;
+            title.AccentColour = colours.Yellow;
```

Comment:
i'd probably make a nested class (`MultiHeaderTitle`) and do this internally. we generally avoid doing this kind of cross-class operation.